src/DevBetterWeb.Infrastructure/PaymentHandler/StripePaymentHandler/StripeCustomer.cs

```diff
@@ -0,0 +1,28 @@
+﻿using DevBetterWeb.Core.Interfaces;
+using Stripe;
+
+namespace DevBetterWeb.Infrastructure.PaymentHandler.StripePaymentHandler
+{
+  public class StripeCustomer : IPaymentHandlerCustomer
```

Comment:
This is a service or a handler (which is a special kind of service), so its name should have a suffix reflecting this.